### ISSUE TITLE
default country Error  during migration

### ISSUE THREAD

**Issue by A-linea** [remp2020/crm-skeleton#11]
Hello! After composer update,  I tried repeating steps 7-11 and an error occurred during the migration.

![CleanShot 2023-10-16 at 15 25 39](https://github.com/remp2020/crm-skeleton/assets/17249894/2d5766c2-ef81-4b41-bcf5-5ab467980153)

**Comment by rootpd** [comment]
Thanks for the report, we'll have to fix this. The DB migration that crashed relied on something, that wasn't ready yet.

Can you try to run `application:seed` command first and then go through 7-11 again? That should help until we issue a hotfix.

**Comment by A-linea** [comment]
Thanks for fast response. I find another issue, at this moment I added country by myself to db as temp sollution, but now, no matter what I do, I get other nasty error.
![CleanShot 2023-10-16 at 15 47 13](https://github.com/remp2020/crm-skeleton/assets/17249894/ac19905f-e48f-4c50-b708-86a10afdcdab)

Upd:
Unfortunately, I have no idea why it appears only in my working project, I tried to do a clean installation in another directory, and made a small fix in the database (filled countries table), and all further steps worked as expected. Maybe later it will be easier to just migrate the modules while it's local development.

UPD2: At this moment i solved second issue by copied your composer.lock from repository.
